src/redux/auth/saga.ts

```diff
@@ -5,9 +5,10 @@ import * as Google from 'expo-google-app-auth';
 import decode from 'jwt-decode';
 import { call, delay, put, takeLatest } from 'redux-saga/effects';
 
-import { ANDROID_CLIENT_ID, FACEBOOK_APP_ID, IOS_CLIENT_ID } from '../../env';
-import { loginUser, signupUser } from '../../utils/apiCalls/authorization';
-import firebase, { myFirebaseApp } from '../../utils/firebase';
+import { ANDROID_CLIENT_ID, FACEBOOK_APP_ID, IOS_CLIENT_ID } from '~~env';
```

Comment:
`'~~env'` it look strange :smile: maybe just `'env'` ?

Reply:
You've got my answer why I wanted to have a `~` prefix :stuck_out_tongue: The prefix is is doubled because it's a level higher than `/src` (maybe to fancy :smile: ). I'll wait for your opinion which prefix do you prefer, but I think it is important just to have some.

Reply:
If `/src` is `~/` so `../src` maybe should be just `../~/`, would it work?